packages/react-scripts/config/webpack.config.js

```diff
@@ -319,9 +322,13 @@ module.exports = function(webpackEnv) {
                 formatter: require.resolve('react-dev-utils/eslintFormatter'),
                 eslintPath: require.resolve('eslint'),
                 // @remove-on-eject-begin
-                baseConfig: {
-                  extends: [require.resolve('eslint-config-react-app')],
-                },
+                baseConfig:
+                  // We allow overriding the config, only if it extends our config
+                  // (`extends` can be a string or array of strings).
+                  appPackageJson.eslintConfig &&
+                  appPackageJson.eslintConfig.extends.includes('react-app')
```

Comment:
`extends` may not be defined.

Reply:
Good point, it was a last minute change from `extends ===` to `extends.includes`.